src/CalcViewModel/DataLoaders/CurrencyDataLoader.cpp

```diff
@@ -9,6 +9,7 @@
 #include "Common/LocalizationSettings.h"
 #include "Common/TraceLogger.h"
 #include "UnitConverterDataConstants.h"
+#include <locale>
```

Comment:
I believe the style of the project is to move standard headers into the pch file. +@danbelcher-MSFT to double check me here.

Reply:
Yes, please move to pch